quickstarts/host-inventory-quickstart.yaml

```
@@ -21,13 +21,31 @@ spec:
 
 
   tasks:
-    - description: >-
+    - description: |-
 
-        1.  Configure your host inventory settings. View [documentation](https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.5/html-single/clusters/index#enable-cim) to follow the steps to configure your host inventory settings.
+        1.  Configure your host inventory settings.
+
+          a. From the **All Clusters** view of the OpenShift Container Platform web console, navigate to the host inventory by clicking infrastructure > **Host inventory**
+
+          b. Select **Configure host inventory settings**.
+
+        1. Enter the required values for **Database storage**, **System Storage**, and **Image Storage** in the modal that opens. 
+
+          a. Enter the required values for Database storage, System Storage, and Image Storage.
+
+          **Note**: The modal is for connected environments only. For disconnected environments, follow the steps for [configuring your host inventory settings documentation](https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.5/html-single/clusters/index#enable-cim).
+
+          b. If your cluster is running on AWS and you have not configured your load balancer, select the 'Configure load balancer on Amazon Web Services for me' checkbox.
+
+        1. Then click configure.
+
+          **Note**: Storage sizes cannot be changed after they are configured. Configuring the load balancer on AWS cannot be reverted.
+        
+        1. The configuration will take time. You will be notified when the configuration is successfully complete. Now your host inventory settings are successfully configured and you can create your first infrastructure environment.
 
         1. Create an infrastructure environment for your host inventory.
 
-          a. Go to **Host inventory**.
+          a. From the **All Clusters** view of the OpenShift Container Platform web console, navigate to the host inventory by clicking infrastructure > **Host inventory**
```

Should it be **Infrastructure**?
```suggestion
          a. From the **All Clusters** view of the OpenShift Container Platform web console, navigate to the host inventory by clicking **Infrastructure** > **Host inventory**
```